lib/testers/rule-tester.js

```diff
@@ -343,35 +353,11 @@ class RuleTester {
                 }
             }));
 
-            // Freezes rule-context properties.
-            const originalGet = linter.rules.get;
-
-            try {
-                linter.rules.get = function(ruleId) {
-                    const originalRule = originalGet.call(linter.rules, ruleId);
-
-                    return {
-                        meta: originalRule.meta,
-                        create(context) {
-                            Object.freeze(context);
```

Comment:
The `Object.freeze(context)` line was removed above because rule contexts have already been frozen since ESLint v2.0.0.